webapp/channels/src/components/sidebar_right/index.ts

```diff
@@ -77,4 +76,4 @@ function mapDispatchToProps(dispatch: Dispatch) {
     };
 }
 
-export default withRouter(connect(mapStateToProps, mapDispatchToProps)(memo(SidebarRight)));
+export default withRouter(connect(mapStateToProps, mapDispatchToProps)(SidebarRight));
```

Comment:
I happened to notice this since I previously had the `SidebarRight` pass down `isMobileView`. I don't think this `React.memo` actually did anything since the `SidebarRight` is already a `PureComponent`